crds/jwst/specs/miri_ipc.rmap

```diff
@@ -0,0 +1,17 @@
+header = {
+    'derived_from' : 'generated as stub rmap on 2014-03-24 15:03:30.015471',
+    'file_ext' : '.fits',
+    'filekind' : 'IPC',
+    'filetype' : 'INTERPIXEL CAPACITANCE',
+    'instrument' : 'MIRI',
+    'mapping' : 'REFERENCE',
+    'name' : 'miri_ipc.rmap',
```

Comment:
Not positive about the names I'm giving them, since other rmaps tend to list out full examples like 
"'name' : 'jwst_nirspec_wavelengthrange_0000.rmap'," for nirspec_wavelengthrange.rmap, 
meanwhile others are simplified like the ones I provided, as in:
"'name' : 'nircam_superbias.rmap'," for nircam_superbias.rmap.

Reply:
At this level, the name is not really relevant. More generic is preferred only because these are just for validation, and the name field is not involved. For a solid rule, it should be the name of the the actual file itself.